### PR TITLE
[RFC] Feature/new backend breadcrumb trail

### PR DESCRIPTION
**Pull request by agoat**
This is the PR to https://github.com/contao/core-bundle/issues/602 to handle unlimited child tables in the breadcumb trail.

It goes recursively through all the parent tables and trys to add the table name and the objects title or name. 
If a table name is shown or not can be controlled whether there is a translation or not.
I also added a suggestion to style the table and the object names differently for a better readability.

Additionally extra translation should be added in the contao/news-bundle to show the tables name (in modules.xlf):
```
      <trans-unit id="MOD.tl_news">
        <source>Articles</source>
      </trans-unit>
```

If someone want to see this in action, I can send the login data for my dev server via email.

### CODE REVIEW

File: src/Resources/contao/classes/Backend.php [hunk 1]
```diff
@@ -479,127 +479,77 @@ protected function getBackendModule($module)
...
-				if ($strTable && isset($GLOBALS['TL_LANG']['MOD'][$strTable]))
+				
+				// Add the last parent table
+				if (isset($GLOBALS['TL_LANG']['MOD'][$table]))
 				{
-					$this->Template->headline .= ' » ' . $GLOBALS['TL_LANG']['MOD'][$strTable];
+					$trail[] = ' &raquo;&nbsp;'. $GLOBALS['TL_LANG']['MOD'][$table];
 				}
+				
+				// Add the beadcrumb trail in reverse order
```
Comment (xchs) [inline src/Resources/contao/classes/Backend.php]: Typo: b**r**eadcrumb

**Comment by leofeyer** [comment]
@contao/developers /cc

**Comment by Toflar** [comment]
Haven't tested it but looks neat! 👍  Thanks for contributing @agoat! I was just thinking about whether this needs to be modifiable/extensible somehow. But I haven't had the use case yet and we could still add an event later on if we want to.

**Comment by aschempp** [comment]
Thanks! A quick code review looks good to me 😎 
There are a few indentation issues (especially in the XLIFF) but these are minor things, @leofeyer will fix the coding style anyway ^^

**Comment by agoat** [comment]
Just a quick example of how it look like:
backend-breadcumb-trail1

And a very extreme one using circular parent tables (thats why I have made thid PR):
backend-breadcumb-trail2

**Comment by leofeyer** [comment]
Thank you @agoat. 👍

**Pull request merged**